src/ior.c

```diff
@@ -1309,7 +1309,7 @@ static void TestIoSys(IOR_test_t *test)
 
                 FillBuffer(ioBuffers.buffer, params, 0, pretendRank);
                 /* use repetition count for number of multiple files */
-                if (params->multiFile)
```

Comment:
can you please explain this?

Reply:
The original code seemed to use the iteration count only for the MULTI driver.  But I need this count to know during which iteration I can inject the fault.